notebooks/Lab8.ipynb

```diff
@@ -108,7 +126,7 @@
     "editable": true
    },
    "source": [
-    "So we have 110 patients in our trial."
+    "So, there are 88 patients who have `Chromosome` equal to `21`."
```

Comment:
There should be 110 patients. "who have `Chromosome` equal to `21`." is misleading.